workflow-ui/core-android/src/androidTest/java/com/squareup/workflow1/ui/OnBackPressedDispatcherSpy.java

```diff
@@ -0,0 +1,15 @@
+package androidx.activity;
+
+import java.util.ArrayDeque;
+
+public class OnBackPressedDispatcherSpy {
+  private final OnBackPressedDispatcher dispatcher;
+
+  public OnBackPressedDispatcherSpy(OnBackPressedDispatcher dispatcher) {
+    this.dispatcher = dispatcher;
+  }
+
+  public ArrayDeque<OnBackPressedCallback> callbacks() {
+    return dispatcher.mOnBackPressedCallbacks;
```

Comment:
nit: Collection or List maybe?

Reply:
I'm exposing the implementation of an Android class for testing purposes, I don't think there's a real need to abstract it.